pkg/pillar/cmd/volumemgr/handlevolume.go

```diff
@@ -26,7 +65,27 @@ func handleVolumeModify(ctxArg interface{}, key string,
 	log.Infof("handleVolumeModify(%s)", key)
 	config := configArg.(types.VolumeConfig)
 	ctx := ctxArg.(*volumemgrContext)
-	updateVolume(ctx, config)
+	status := lookupVolumeStatus(ctx, config.Key())
```

Comment:
Can't this return nil? You're not checking for nil.

Reply:
Resolved.